test/testinput/dirac.yml

```
@@ -1,21 +1,44 @@
-background error:
-  covariance model: umdsstCovar
-  horizontal_length_scale: 2.2e6
-  maximum_condition_number: 1.0e6
-  standard_deviation: 1.8e7
-dirac:
-  date: 2010-01-01T12:00:00Z
-  ixdir: [20, 50, 100, 150, 200, 250, 300, 350]
-  iydir: [10, 25, 50,  75,  100, 125, 150, 175]
 geometry:
   grid: S360x180
   domain:
     type: global
     west: -180
+
+analysis variables: &vars [sea_surface_temperature]
+
+background:  # must have
+  state variables: *vars
+  date: 2018-04-15T00:00:00Z
+  filename: Data/19850101_regridded_sst.nc
+
+background error:
+  covariance model: BUMP
+  bump:
+    verbosity: main
+    datadir: Data
+    forced_radii: 1
+    method: cor
+    new_nicas: 1
+    ntry: 3
+    prefix: Chen
+    resol: 6.0
+    mpicom: 2
+    rh:
+      sea_surface_temperature: [3000.0e3]
+    rv:
+      sea_surface_temperature: [1]
+    strategy: specific_univariate
+
```

`staticbinit.x` is responsible for creating the static B, after that  `dirac.x` and `var.x` should load that static B in, so it doesn't need to be recreated here (`load_nicas: 1`)

```suggestion
background error:
  covariance model: BUMP
  bump:
    verbosity: main
    datadir: Data
    method: cor
    load_nicas: 1
    prefix: Chen
    mpicom: 2
```

---

We need to keep "strategy: specific_univariate", otherwise it'll failed with error "!!! ABORT in nam_check on task #0002: wrong strategy"